modules/test/playwright/tests/frontend-data-set-views-web/pages/ViewsPage.ts

```diff
@@ -57,18 +51,23 @@ export class ViewsPage {
 		await this.newDataSetViewModal.saveButton.click();
 	}
 
-	async gotoDataSetView(name = DEFAULT_DATA_SET_VIEW_NAME) {
+	async goto(dataSetLabel: string = DEFAULT_LABEL.DATA_SET) {
+		await this.dataSetsPage.goto();
+		await this.dataSetsPage.gotoDataSet(dataSetLabel);
+	}
+
+	async openDataSetView(viewLabel: string = DEFAULT_LABEL.VIEW) {
```

Comment:
I renamed this from `goto*` to `open*`. I am suggesting here a naming convention:
- `goto*` implies navigation from home page to the end page.
- any other navigation local in page should be named something else, `open*`, `select*` etc.